cea/utilities/epwreader.py

```diff
@@ -27,8 +28,9 @@ def epw_reader(weather_path):
                   'snowdepth_cm', 'days_last_snow', 'Albedo', 'liq_precip_depth_mm', 'liq_precip_rate_Hour']
 
     result = pd.read_csv(weather_path, skiprows=8, header=None, names=epw_labels).drop('datasource', axis=1)
-    result['date'] = pd.Series(pd.date_range(str(result["year"][0])+"/1/1", periods=8760, freq='H'))
-    result['dayofyear'] = pd.date_range(str(result["year"][0])+"/1/1", periods=8760, freq='H').dayofyear
+    result = result.loc[0:8759]
```

Comment:
um... to be consistent, this should actually be `result.loc[0:HOURS_IN_YEAR-1]`, right?

Reply:
You're right. Done!